### DISCUSSION TITLE
Option to disable feed

### DISCUSSION THREAD

**Discussion by richard-barella** [dailydotdev/daily#1334] [Feature Requests]
Feeds are distracting for me. I want to have daily.dev as my homepage but without seeing the feed everytime.

**Comment by idoshamun** [comment]
Hi @ribab,

Can you elaborate what you expect if you keep daily.dev as your homepage but without the feed?

**Reply by richard-barella** [reply]
@idoshamun yeah maybe you are right. there isn't much benefit to disabling the feed and keeping daily.dev as the home page.
